### PR TITLE
Fixes #15309: at install on the server on centos7, rudder-system-directives.cf is empty, and so, there are missing bundles when running the agent

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15309

### CODE REVIEW

File: techniques/system/common/1.0/promises.st [hunk 1]
```diff
@@ -146,12 +146,13 @@ bundle common va
...
+    # all other inputs are loaded by body file control in rudder-system-directives.cf
       "inputs_list" slist => { @{ncf_inputs} };
 
 &if(INITIAL)&
+      # rudder-directives.cf is needed, even if empty since we call it in the bundle sequence
       # Idealy these should be generated at compile time from metadata.xml and put into variable.json
-      "common_input_list" slist => { "common/1.0/common.cf", "common/1.0/cf-serverd.cf", "common/1.0/rudder-groups.cf", "common/1.0/hooks.cf", "common/1.0/cron-setup.cf", "common/1.0/site.cf", "common/1.0/update.cf", "common/1.0/monitoring.cf", "common/1.0/restart-services.cf", "common/1.0/internal-security.cf",  "common/1.0/environment-variables.cf", "common/1.0/properties.cf" };
+      "common_input_list" slist => { "rudder-directives.cf", "common/1.0/common.cf", "common/1.0/cf-serverd.cf", "common/1.0/rudder-groups.cf", "common/1.0/hooks.cf", "common/1.0/cron-setup.cf", "common/1.0/site.cf", "common/1.0/update.cf", "common/1.0/monitoring.cf", "common/1.0/restart-services.cf", "common/1.0/internal-security.cf",  "common/1.0/environment-variables.cf", "common/1.0/properties.cf" };
```
Comment (amousset) [inline techniques/system/common/1.0/promises.st]: this file is not part of the common technique, I is probably not the right place

Comment (peckpeck) [reply techniques/system/common/1.0/promises.st]: it is

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**